src/components/DataDimension/Groups.js

```diff
@@ -31,11 +32,7 @@ export const Groups = ({
             <div className="group-container">
                 <SingleSelectField
                     label={dataTypes[dataType].getGroupLabel()}
-                    selected={
-                        selected.id && selected.name
-                            ? { value: selected.id, label: selected.name }
-                            : {}
-                    }
+                    selected={selected.id && selected.name ? selected.id : null}
```

Comment:
Probably only need to check `selectedId` ?

Reply:
Yes, actually `groupId` can be used directly